src/pages/apps/index.jsx

```diff
@@ -0,0 +1,238 @@
+    .sort((a, b) => {
+      if (activeSort === SORTS.ALPHABETICAL) {
+        return a.name.localeCompare(b.name);
+      } else if (activeSort === SORTS.RECENT) {
+        return new Date(a.date) > new Date(b.date) ? -1 : 1;
+      }
+    })
+    .filter((app) => {
+      if (activeQuery && activeQuery !== '') {
+        console.log(activeQuery, app.name.toLowerCase());
+        return app.name.toLowerCase().includes(activeQuery);
+      } else {
+        return true;
+      }
+    });
+
+  const filters = Object.entries(categories);
+
+  const renderSort = (sort) => {
+    return (
+      <li
+        key={sort}
+        className={clsx(
+          'dropdown__link',
+          activeSort === sort && 'dropdown__link--active',
+          styles.sortDropdownItem
+        )}
+        onClick={() => setActiveSort(sort)}
+      >
+        <span>{sort}</span>
+      </li>
+    );
+  };
+
+  const renderPillFilter = (filter) => {
+    const [name, entries] = filter;
+    return (
+      <li
+        key={name}
+        className={clsx(
+          'pills__item',
+          styles.pillFilter,
+          activeCategory && activeCategory[0] === name && 'pills__item--active'
+        )}
+        onClick={() => setActiveCategory(filter)}
+      >
+        <span>{name}</span>
+        <span className={clsx('badge badge--secondary', styles.filterBadge)}>
+          {entries.length}
+        </span>
+      </li>
+    );
+  };
+
+  const renderDropdownFilter = (filter) => {
+    const [name, entries] = filter;
+    return (
+      <li
+        key={name}
+        className={clsx(
+          'dropdown__link',
+          styles.dropdownFilterItem,
+          Array.isArray(activeCategory) &&
+            activeCategory[0] === name &&
+            'dropdown__link--active'
+        )}
+        onClick={() => setActiveCategory(filter)}
+      >
+        <span>{name}</span>
+        <span className={clsx('badge badge--secondary', styles.filterBadge)}>
+          {entries.length}
+        </span>
+      </li>
+    );
+  };
+
+  const currentFavs = activeCategory
+    ? activeCategory[1].filter((app) => app.isFavorite)
+    : sortedApps.filter((app) => app.isFavorite);
+
+  return (
+    <Layout title="App Showcase">
+      <main className="margin-vert--xl">
+        <h1 className={styles.title}>Showcase</h1>
+        <p className={styles.subtitle}>
+          Discover <strong>hundreds of production applications</strong> built
+          with Electron.
+        </p>
+        <div className={clsx('margin-bottom--md', styles.filtersContainer)}>
+          <div
+            className={clsx(
+              'container',
+              'margin-bottom--sm',
+              styles.sortAndSearchContainer
+            )}
+          >
+            <div className="dropdown dropdown--hoverable">
+              <button className="button button--secondary">
+                Sort: {activeSort}
+              </button>
+              <ul className="dropdown__menu">
```

Comment:
```suggestion
              <ul className={clsx('dropdown__menu', styles.sortDropdownMenu)}>
```

```scss
// File: src/pages/apps/apps.module.scss
.sortDropdownMenu {
  min-width: 100%;
}
```

Before:
<img width="223" alt="image" src="https://user-images.githubusercontent.com/8198408/186058404-430f6509-d145-4b1c-b5f3-1d09ebbba284.png">


After:
<img width="230" alt="image" src="https://user-images.githubusercontent.com/8198408/186058548-81d1d65c-ced6-4d96-93d4-c188c4657755.png">